res/layout/generic_list.xml

```diff
@@ -1,7 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
-<com.gh4a.widget.ContextMenuAwareRecyclerView
+<FrameLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/list"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:scrollbars="vertical" />
+    android:layout_height="match_parent">
+
+    <com.gh4a.widget.ContextMenuAwareRecyclerView
+        android:id="@+id/list"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent" />
+
+    <com.pluscubed.recyclerfastscroll.RecyclerFastScroller
+        android:id="@+id/fast_scroller"
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        android:layout_gravity="end"
```

Comment:
right or right|end

(min API is < 17 and we don't have RTL language translations anyway)